grip_api/src/grip_api/task_editor_widgets/container.py

```diff
@@ -50,20 +50,20 @@ def __init__(self, editor_widget, container_type):
         self.graphics_container = GraphicsContainer(self)
         self.graphics_container.set_graphics_scene(64000, 64000)
         # Store the different states present in the container
-        self.states = list()
+        self.states = []
         # Get specific list of states that need to be generated
-        self.states_to_generate = list()
+        self.states_to_generate = [[]]
```

Comment:
wtf

Reply:
why list of list?

Reply:
Likely a misclick, changed [here](https://github.com/shadow-robot/sr_grip/pull/16/commits/e8ac01c3bac38f78ed47b322574ebdca12c6bd2f)